pkg/kubectl/cmd/create.go

```diff
@@ -183,13 +183,12 @@ func RunCreate(f cmdutil.Factory, cmd *cobra.Command, out, errOut io.Writer, opt
 		return err
 	}
 
-	mapper, typer, err := f.UnstructuredObject()
+	mapper, _, err := f.UnstructuredObject()
 	if err != nil {
 		return err
 	}
 
-	r := f.NewBuilder().
-		Unstructured(f.UnstructuredClientForMapping, mapper, typer).
+	r := f.NewUnstructuredBuilder().
```

Comment:
We removed this in a previous PR because a builder needs to be aware of both typed and unstructured content.  I'm going to revert this particular change in my PR because we want to move to a model where a single builder can deal with all types.

Reply:
Specifically in 90d76adb4bf1180782528b10852d2a12ce35022d